packages/components/src/components/Indicator/Indicator.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+import { spacingsPx, spacings } from '@trezor/theme';
+
+import { Box } from '../Box/Box';
+
+const Circle = styled.div<{ $isActive?: boolean }>`
+    width: ${spacingsPx.xs};
+    height: ${spacingsPx.xs};
+    border-radius: 50%;
+    background: ${({ theme, $isActive }) =>
+        $isActive ? theme.backgroundPrimaryDefault : theme.backgroundNeutralSubdued};
+    transition:
+        background 0.5s,
+        outline 0.5s;
+    outline: ${({ theme, $isActive }) =>
+        `${spacingsPx.xxs} solid ${$isActive ? theme.backgroundPrimarySubtleOnElevation0 : 'transparent'}`};
+`;
+
+export type IndicatorProps = {
+    isActive?: boolean;
+};
+
+export const Indicator = ({ isActive }: IndicatorProps) => (
```

Comment:
What about rename it to `DotIndicator` or `DotStatus`? It's hard to imagine what it is from current name. WDYT?